src/tunacode/tools/lsp/diagnostics.py

```
@@ -4,41 +4,36 @@
-    return {}
+        return DEFAULT_USER_CONFIG["settings"]["lsp"]
+    return config["settings"]["lsp"]
 
 
 def is_lsp_enabled() -> bool:
     """Check if LSP diagnostics are enabled in user config."""
     try:
-        lsp_config = _get_lsp_settings()
-        return bool(lsp_config.get("enabled", False))
+        return _get_lsp_settings()["enabled"]
     except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)